recipes/continuable/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.11)
```

```suggestion
cmake_minimum_required(VERSION 2.8.12)
```

Don't need Cmake 3+ for this simple file.